src/sonic-yang-models/yang-models/sonic-buffer-pool.yang

```diff
@@ -57,6 +61,17 @@ module sonic-buffer-pool {
                     type uint64;
                     description "Buffer Pool Xoff Threshold (in Bytes)";
                 }
+
+                leaf percentage {
+                    type uint8;
```

Comment:
Please have a range of 0-100 for percentage

Reply:
I would like to allow a user to oversubscribe the buffer (eg. for debugging purposes). this is why I didn't put an upper bound here. as it is unsigned type, lower bound is not required either.
how do you think?